### ISSUE TITLE
Error in full tour with Prometheus tutorial

### ISSUE THREAD

**Issue by bacherfl** [keptn/tutorials#91]
When uploading the sli-config for the quality gates for the carts service in the staging stage, the following sli-config is used (see [here](https://tutorials.keptn.sh/tutorials/keptn-full-tour-prometheus-07/index.html?index=..%2F..index#13):

```
---
spec_version: '1.0'
indicators:
  response_time_p50: histogram_quantile(0.5, sum by(le) (rate(http_response_time_milliseconds_bucket{handler="ItemsController.addToCart",job="$SERVICE-$PROJECT-$STAGE"}[$DURATION_SECONDS])))
  response_time_p90: histogram_quantile(0.9, sum by(le) (rate(http_response_time_milliseconds_bucket{handler="ItemsController.addToCart",job="$SERVICE-$PROJECT-$STAGE"}[$DURATION_SECONDS])))
  response_time_p95: histogram_quantile(0.95, sum by(le) (rate(http_response_time_milliseconds_bucket{handler="ItemsController.addToCart",job="$SERVICE-$PROJECT-$STAGE"}[$DURATION_SECONDS])))
```

In this config, the job selector is set to `job="$SERVICE-$PROJECT-$STAGE"`, which means that the returned SLI values will be the ones of the current primary deployment, and not the one that is tested currently.

When skipping the upload of the `sli-config-prometheus.yaml`, the use case works again, since the prometheus-sli-service will fall back to the default queries, which select the current canary deployment per default.

**Comment by jetzlstorfer** [comment]
Thanks @bacherfl for bringing this up.
However, to make it more obvious how SLIs work in Keptn we should keep this step instead of removing it and falling back to the default ones.
Can you please provide the SLIs of the fallback version so we can change this in the file? 

Thanks!

**Comment by jetzlstorfer** [comment]
For reference, here is the file that has to be fixed:
https://github.com/keptn/examples/blob/master/onboarding-carts/sli-config-prometheus.yaml

**Comment by jetzlstorfer** [comment]
Addressed in #105
